client/web/src/components/externalServices/ExternalServiceForm.tsx

```diff
@@ -37,8 +42,12 @@ interface Props extends Pick<AddExternalServiceOptions, 'jsonSchema' | 'editorAc
     autoFocus?: boolean
     externalServicesFromFile: boolean
     allowEditExternalServicesWithFile: boolean
+    additionalFormComponent?: ReactNode
 }
 
+const ajv = new AJV({ strict: false, $comment: true })
```

Comment:
maybe? this way it won't require a reader to know that AJV is a JSON validation library
```suggestion
const jsonSchemaValidator = new AJV({ strict: false, $comment: true })
```

Reply:
This is convention in other parts of the code, so I'll keep it. In the end, it's just variable name, so not super important.

Reply:
in this case I'm fine with this 👍🏻